lib/split/dashboard/views/_controls.erb

```diff
@@ -1,3 +1,15 @@
+<% if experiment.cohorting_disabled? %>
```

Comment:
We can hide this button if the experiment is finished?

Reply:
👍